src/NServiceBus.Core/Pipeline/Outgoing/RoutingToDispatchConnector.cs

```diff
@@ -52,7 +40,34 @@ public override Task Invoke(IRoutingContext context, Func<IDispatchContext, Task
             return stage(this.CreateDispatchContext(operations, context));
         }
 
+        static void LogOutoingOperationsWhenDebugEnabled(TransportOperation[] operations)
+        {
+            if (!isDebugEnabled)
```

Comment:
That was originally my intention. Learned something since I thought static is enough.